front/app/utils/helperUtils.test.ts

```diff
@@ -103,3 +104,37 @@ describe('isPage', () => {
     });
   });
 });
+
+describe('isTopBarNavActive', () => {
+  it('returns true when pathname and tabUrl end with basePath', () => {
+    const basePath = '/dashboard';
+    const pathname = '/user/dashboard';
+    const tabUrl = '/user/dashboard';
+    const result = isTopBarNavActive(basePath, pathname, tabUrl);
+    expect(result).toBe(true);
+  });
+
+  it('returns false when only tabUrl ends with basePath', () => {
```

Comment:
```suggestion
  it('returns false when tabUrl but not pathname ends with basePath', () => {
```

Let's be explicit? Makes it easier to follow. :-)